modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Editor/Editor.tsx

```diff
@@ -52,7 +52,14 @@ import CodeMirror from 'codemirror';
 
 import './Editor.scss';
 
-export default function Editor({content, disabled, setValues}: EditorProps) {
+import './Sidebar/Sidebar.scss';
+
+export default function Editor({
+	content,
+	disabled,
+	inputChannel,
+	setValues,
+}: EditorProps) {
 	const defaultContent = `<#-- Insert a Groovy Script to define your validation. -->`;
```

Comment:
@cgmonte @MariMalta You will need to display different texts for Groovy and DDM Expression Builder. Make sure to use the texts provided in the mockup.